wagtail/wagtailadmin/templates/wagtailadmin/block_forms/struct.html

```diff
@@ -5,9 +5,9 @@
 
     <ul class="fields">
         {% for child in children.values %}
-            <li>
+            <li{% if child.block.field.required %} class="required"{% endif %}>
```

Comment:
We probably want to avoid referring to `child.block.field` here - block objects don't always wrap a Django form field, and the ones that don't might have a different idea of how "required-ness" is defined. I think we can fix this nicely by adding a standard `required` property to Block (which would be false by default, and pick up the value from `self.field` in the case of FieldBlock) - I'll make this change.